src/gridui.cpp

```diff
@@ -117,12 +117,18 @@ bool _GridUi::handleRbPacket(const std::string& cmd, rbjson::Object* pkt) {
         if (changed) {
             m_states_modified = true;
         }
+        m_tab_changed = true;
     } else {
         return false;
     }
     return true;
 }
 
+void _GridUi::changeTab(uint16_t index) {
+    m_tab_changed = true;
```

Comment:
Pozor, sem a do `stateChangeTask` zase ten mutex patří, protože tam je více proměnných - jen nahoře v `handleRbPacket` se pouze nastavuje jeden atomic bool, takže tam je to ok.